torchinductor/scheduler.py

```diff
@@ -418,6 +422,105 @@ def get_name(self):
         return self.node.get_name()
 
 
+def get_fake_func(name):
+    def func1(*args):
+        return 0
+
+    func1.__name__ = name
+    return func1
+
+
+def create_fx_graph(nodes, fname, print_graph=False):
```

Comment:
By the way, let's move this code to `utils.py`

Reply:
resolved offline. Will keep this here because moving it to utils.py might cause circular dependency in the future.

Reply:
Also, let's rename this to `create_fx_from_buffers`

Reply:
renamed